Skipping_Test_Files/output.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file looks like a test output file. Please remove it from GIT history, we probably not want this in GIT logs.